dev-python/virtualenvwrapper/virtualenvwrapper-4.8.4_p20230121.ebuild

```
@@ -0,0 +1,68 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..11} )
+
+inherit distutils-r1
+
+DESCRIPTION="Set of extensions to Ian Bicking's virtualenv tool"
+HOMEPAGE="
+	https://github.com/python-virtualenvwrapper/virtualenvwrapper
+	https://pypi.org/project/virtualenvwrapper/
+"
+
+COMMIT="e63d2c389ed276aa161bb50a27c77af94d96a50c"
+SRC_URI="https://github.com/python-virtualenvwrapper/virtualenvwrapper/archive/${COMMIT}.tar.gz -> ${P}.gh.tar.gz"
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+export PBR_VERSION="$(ver_cut 1-3).post$(ver_cut 5-)"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+RDEPEND="
+	$(python_gen_cond_dep '
+		dev-python/virtualenv[${PYTHON_USEDEP}]
+		dev-python/stevedore[${PYTHON_USEDEP}]
+		dev-python/virtualenv-clone[${PYTHON_USEDEP}]
+	')
+"
+BDEPEND="
+	${RDEPEND}
+	$(python_gen_cond_dep '
+		dev-python/pbr[${PYTHON_USEDEP}]
+	')
+"
+
+src_prepare() {
+	default
+
+	# specify default python interpeter to align with PYTHON_SINGLE_TARGET
+	sed -i -e \
+		"s|_virtualenvwrapper_python_executable=\"\$(.*)\"|_virtualenvwrapper_python_executable=\"${PYTHON}\"|" \
+		"${S}/virtualenvwrapper.sh" || die
```

```suggestion
		virtualenvwrapper.sh || die
```

S is the current directory, right?